CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+# Contributing to Newsboat
+
+Thank you for your interest in contributing to Newsboat! Here are a few ways
+you can help:
+
+1. [Feature Requests](#feature-requests)
+2. [Bug Reports](#bug-reports)
+3. [Patches / Pull Requests](#patches--pull-requests)
+    1. [Testing](#testing)
+    2. [Documentation](#documentation)
+    3. [Translations](#translations)
+
+## Feature Requests
+
+Feature requests should be reported in the
+[issue tracker](https://github.com/newsboat/newsboat/issues) or on
+the [mailing list](https://groups.google.com/group/newsboat). Ensure
+the feature request was not already reported by searching the issue tracker
+and/or mailing list. Feature requests are filed under the `enhancement` label
+on the [issue tracker](https://github.com/newsboat/newsboat/labels/enhancement).
+
+
+## Reporting Bugs
+
+**Do not open a GitHub issue if the bug is a security vulnerability**. Send an
+email to security@newsboat.org. Please encrypt your emails to 
+[PGP key 4ED6CD61932B9EBE](https://newsboat.org/newsboat.pgp)
+
+Bug reports should be reported in the
+[issue tracker](https://github.com/newsboat/newsboat/issues) or on
+the [mailing list](https://groups.google.com/group/newsboat). Again,
```

Comment:
As mentioned above, we don't actually track bugs on the mailing list, although we'd accept a report there.